### ISSUE TITLE
Error:  Uncaught ReferenceError: module is not defined 

### ISSUE THREAD

**Issue by sniras** [paypal/PayPal-Cordova-Plugin#42]
Error is generated by the following line number
cdv-plugin-paypal-mobile-sdk.js:146 (anonymous function)
which is as follows:

 module.exports = new PayPalMobile(); //line that causes error

**Comment by rknell** [comment]
Hello, I had a similar problem and have been tearing my hair out - It seemed that SOMETIMES a build would work - and others it would just error out.

The trick is, they have now made the cdv-plugin.....js file an actual cordova plugin - make sure you remove it from your index.html as it gets injected by the cordova system. When its in the index.html you will get the module error, and sometime it will work if cordova attaches the data after your index version (the functions were being overridden!)

TL;DR Dont reference the cordova plugin js in your index.html

**Comment by romk1n** [comment]
@rknell you right the only file one need to include is `paypal-mobile-js-helper.js` helper file as per integration instructions https://github.com/paypal/paypal-cordova-plugin#your-app-integration

**Comment by romk1n** [comment]
since there is no response from @sniras i am going to close this issue. please feel free to open it again if those tips didn't help.

**Comment by jguix** [comment]
Actually it doesn't seem to be necessary to copy the file from the plugin location to the www folder, following the sample application you only have to include the reference like this:

    
    <script src="js/paypal-mobile-js-helper.js"></script>
